### DISCUSSION TITLE
fa/manager/configuration-and-advanced-settings/How-to-activate-WARP-on-the-Hiddify-panel/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4193] [Content Help کمک به اصلاح محتوا]
# fa/manager/configuration-and-advanced-settings/How-to-activate-WARP-on-the-Hiddify-panel/

The proxy manager platform.

https://hiddify.com/fa/manager/configuration-and-advanced-settings/How-to-activate-WARP-on-the-Hiddify-panel/

**Comment by zekerya** [comment]
===========================================================
####75####Applying...####Warp####
===========================================================
===run.sh other/warp
Checking WARP ...
Checking WARP ...
mv: cannot stat 'wgcf-account.toml': No such file or directory
Checking WARP ...
mv: cannot stat 'wgcf-account.toml': No such file or directory
Generating WARP config...
Removing IPV6 from WARP...
Starting WARP.... checking real connectivitiy
WARP is not working!
Checking real connectivitiy again!
WARP is not working!
!!!!!!!!!!!!!!! WARP ERROR

سلام این همه در ایشو گفتیم وارپ مشکل داره چند ماهه هیچکس کاری نکرد. هیچ جوره وصل نمیشه اصلا کل
 کانفیگها با روشن کردن وارپ از کار میفتن

**Reply by TomCat-Hub** [reply]
درود بر شما
وقت بخیر

خیلی ممنونم که کامنت گذاشتید و مشکل خودتون رو با ما درمیون گذاشتید. 

متاسفانه با وجود مشکلاتی مثل کمبود نیروی انسانی، کمبود مالی، مشکلات شخصی و حمایت های کم از پروژه، روند توسعه و رفع باگ های کوچک با سرعت کمی انجام میشود و با وجود این مشکلات، انتظاری غیر از این اشتباه است. با این وجود تیم ما تمام تلاش خود را میکند در سریع ترین زمان ممکن گزارش های باگ را برسی و طبقه بندی کند و در نزدیک ترین آپدیت باگ ها را رفع کند.

در حال حاضر گزارش زیادی از مشکل در اکتیو کردن سرویس وارپ نداشته ایم و در نسخه ی آخر ریلز این سرویس بخوبی اکتیو میشود. لطفا برای فعالسازی سرویس وارپ از لایسنس استفاده نکنید و بدون کلید از آن استفاده کنید.

صمیمانه پیشنهاد میکنم مشکل خودتون را با ذکر جزییات در گروه تلگرام هیدیفای و تاپیک مربوط به منیجر مطرح کنید تا همکارانم برای رفع مشکل شما را راهنمایی کنند.

آدرس گروه تلگرام هیدیفای:
https://t.me/hiddify_board


باتشکر از صبر و انتخاب شما
مانا باشید

**Comment by olivander1** [comment]
سلام خسته نباشین .مشکلی که دارم اینکه وقتی گزینه وارپ در نسخه کامپیوتر فعال میکنم به خوبی سایت ها باز میشن اما برنامه های دیگه مث اپیک گیمز یا تلگرام و ... باز نمیشن اگه میشه یه راهنمایی کنید در این زمینه ممنون

**Reply by TomCat-Hub** [reply]
درود بر شما
وقت بخیر

نرم افزار رو در حالت Run as administrator اجرا کنید و توی نرم افزار هیدیفای و قسمت تنظیمات کانفیگ سرویس مود رو روی حالت VPN بذارید و بعد اپ هیدیفای رو کانکت کنید. پیشنهاد میکنم دایرکت DNS رو هم روی گوگل یا همون 8.8.8.8 بذارید.

لطفا اگر سوال بیشتری داشتید و یا نیاز به راهنمایی بیشتری بود، توی گروه تلگرام هیدیفای پیام بفرستید و مشکل خودتون را با ذکر جزییات بیان کنید تا دوستان شما رو راهنمایی کنند.

باتشکر از شما
مانا باشید!

**Reply by olivander1** [reply]
خیلی ممنون از راهنمایی تون مشکلم حل شد..در گروه های تلگرامتون عضو هستم اما نمی تونم مسیج بزارم ایدی یکی از منیجر هارو امکانش هست بگین تا بهشون اطلاع بدم دسترسی پیام دادن رو برام باز کنن. تشکر

**Reply by TomCat-Hub** [reply]
درودی دوباره
وقت بخیر

متاسفانه امکان ارائه آیدی مدیران گروه به شما فراهم نیست. اکانت شما برای عضویت در گروه تلگرام هیدیفای باید یک یوزرنیم معتبر داشته باشد. اگر این پیش شرط فراهم است، بعد از عضویت در گروه به تاپیک اطلاعیه رفته، قوانین گروه را با دقت بخوانید و با آن ها موافقت کنید. در این صورت هیچگونه محدودیتی برای شروع فعالیت در چهارچوب قوانین گروه ندارید.

اما اگر از قبل عضو گروه بوده اید و بعلت نقض قوانین از فرستادن پیام در گروه محروم شده اید، متاسفانه راهی برای رفع محدودیت نیست و لازم است تا پایان مدت جریمه صبر داشته باشید.
قویا و دوستانه توصیه میکنم به قوانین ما احترام بگذارید و در ایجاد فضایی امن برای فعالیت سازنده در راستای سیاست های هیدیفای ما را یاری کنید. هدف ما امنیت شما کاربران عزیز و حفظ نظم گروه است.

باتشکر از صبر و شکیبایی شما
مانا باشید

**Comment by olivander1** [comment]
سلام و خسته نباشید ..در اخرین نسخه کامپیوتر من زمانی که vpn  رو run  میکنم به پروکسی بدون مشکل متصل میشه و ping هم میده اما دقیقا زمانی که وصل میشه اتصال اینترنتم قطع میشه و وقتی اتصال vpn رو قطع می کنم نتم وصل میشه ..لطفا اگه امکانش هست یه راهنمایی بکنید خیلی ممنون

**Comment by Sajjad-Arjmand** [comment]
سلام. من وارپ رو بدون مشکل میتونم فعال کنم منتهی لوکیشن سرور من فنلاند هست و به وارپ هنگ کنگ وصل میشه نمیدونم چیکارش کنم. راه دیگه ای نیست که بشه محدودیت ها رو برطرف کرد به جز استفاده از وارپ؟

**Comment by hiwasam** [comment]
سلام
دستور تغییر ایپی رو‌ میزنم فقط ۳ تا ایپی هستن و بین اینا میچرخه 
چطور میشه به ایپی های بیشتری دسترسی داشت؟

**Reply by Sajjad-Arjmand** [reply]
نمیشه اون اسکریپت خیلی محدوده

**Comment by alin30511** [comment]
سلام و خسته نباشید
مشکل نصب وارپ رو دارم همش ارور میده 

Reading package lists... Done
E: The repository 'https://packagecloud.io/ookla/speedtest-cli/ubuntu noble Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package wireguard-dkms
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package wireguard-dkms
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package wireguard-dkms

با این ارور روبه رو میشم لطفا راهنمایی کنین
